### PR TITLE
Removing ALPHA from basic predictors

### PR DESCRIPTION
**Pull request by maxhutch**
# Citrine Python PR

## Description 
The predictor system, graph predictor, expression predictor,
and simple ml predictor are out of alpha.

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

### Adherence to team decisions
- [ ] I have added tests for 100% coverage
- [ ] I have written Numpy-style docstrings for every method and class.
- [ ] I have communicated the downstream consequences of the PR to others.
- [ ] I have bumped the version in setup.py

### CODE REVIEW

File: docs/source/workflows/predictors.rst [hunk 1]
```diff
@@ -288,8 +288,8 @@ The following example illustrates how an :class:`~citrine.informatics.predictors
         training_data=[data_source]
     )
 
-Simple mixture predictor
-------------------------
+Simple mixture predictora (ALPHA)
```
Comment (maxhutch) [inline docs/source/workflows/predictors.rst:291]:
```suggestion
Simple mixture predictor (ALPHA)
```

File: tests/informatics/test_descriptors.py [hunk 2]
```diff
@@ -45,12 +44,6 @@ def test_invalid_eq(descriptor):
     assert not descriptor == other
 
 
-def test_inorganic_deprecated():
```
Comment (maxhutch) [inline tests/informatics/test_descriptors.py:48]: Removed since `InorganicDescriptor` is being removed

**Review by maxhutch — COMMENTED** [review]
Comments from chats with @alexandraj777

**Review by alexandraj777 — APPROVED** [review]
re: offline conversation, approved pending typo fixes

**Pull request merged**